### ISSUE TITLE
[Sev2] Screen reader is not announcing the associated group label “Graph tooltip” for the Default radio button under Panel option control

### ISSUE THREAD

**Issue by JonCole** [grafana/grafana#117285]
User Experience:

Users relying on screen readers (NVDA, JAWS, Narrator) cannot understand the purpose and behavior of the Default radio button because the associated group label and descriptive information are not announced, which impacts comprehension of the Panel option settings.
Note:

**Steps to reproduce**:
1. elect New Dashboard.
2. Navigate to "Add visualization" button and hit ENTER key. "Select data source" popup will appear.
3. Navigate to the TestData list item select it.
4. Now navigate to the tool bar and select back to dashboard control and select. Now select setting control in the tool bar.
5. Navigate to all General tab  under Settings blade.
6. Verify whether Screen reader is announcing the associated group label “Graph tooltip” for the Default radio button under Panel option control. or not.

**Actual Result**:
Screen reader is not announcing the associated group label “Graph tooltip” for the Default radio button under Panel option control.
Observation: 
Screen reader announcement (NVDA / JAWS / Narrator): “Default, radio button, checked, 1 of 3”
The associated group label and description “Graph tooltip. Controls tooltip and hover highlight behavior across different panels. Reload the dashboard for changes to take effect” are not announced.

**Expected Result**
Screen readers (NVDA, JAWS, Narrator) should announce the associated group label and description along with the radio button, for example:
“Graph tooltip. Controls tooltip and hover highlight behavior across different panels. Default, radio button, checked, 1 of 3.”

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
WCAG: [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
Section 508: [E207.2 WCAG Conformance](https://www.access-board.gov/ict/#E207.2), [E205.4 Accessibility Standard](https://www.access-board.gov/ict/#E205.4) and [E205.4.1 Word Substitution when Applying WCAG to Non-Web Documents](https://www.access-board.gov/ict/#E205.4.1), and [E207.2.1 Word Substitution when Applying WCAG to Non-Web Software](https://www.access-board.gov/ict/#e207-2-1-word-substitution-when-applying-wcag-to-non-web-softwar).
EN 301 549: [PDF of EN 301 549](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf), which applies WCAG principles for Info and Relationships to 9.1.3.1 web, 10.1.3.1 non-web documents, and 11.1.3.1.1 software.

**Environment**:
- Grafana version: 12.2.0
- Data source type & version: Any (TestData is what we used)
- User OS & Browser: Chrome or Edge
- Others:
